src/components/overview/sub-Components/StyleSelection.jsx

```diff
@@ -40,19 +40,22 @@ const StyleSelection = function(props) {
     let row = props.state.allStyles.slice(index, index + 4);
     styleRows.push(row);
   }
-
+  var outerkey = 0;
+  var innerkey = 0;
   return (
     <div>
       <div style={{ fontSize: '15pt' }}>
         SELECTED STYLE --> {props.state.selectedStyle.selectedStyleColor}
       </div>
       <div style={containerStyle}>
         {styleRows.map(eachStyleRow => {
+          outerkey++;
```

Comment:
Having a hard time understanding how the outerkey and innerkey functionality comes into play. Possibly look into re-labeling some of this code to make it easier to follow at points in this file.